src/jit/emitxarch.cpp

```diff
@@ -2375,7 +2378,7 @@ void                emitter::emitIns(instruction ins)
             ins == INS_stosb   ||
             ins == INS_stosd   ||
             ins == INS_stosp 
-#ifdef _TARGET_AMD64_
+#ifndef LEGACY_BACKEND
```

Comment:
I'm not sure what the right approach is here, but because most of the "AVX" instructions are simply re-encodings of the SSE* instructions, there isn't a lot of distinction at the `instruction